modules/exploits/multi/http/navigate_cms_rce.rb

```diff
@@ -0,0 +1,128 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => 'Navigate CMS Unauthenticated Remote Code Execution',
+      'Description' => %q(
+        This module exploits insufficient sanitization in the database::protect
+        method, of Navigate CMS versions 2.8 and prior, to bypass authentication.
+
+        The module then uses a path traversal vulnerability in navigate_upload.php
+        that allows authenticated users to upload PHP files to arbitrary locations.
+        Together these vulnerabilities allow an unauthenticated attacker to
+        execute arbitrary PHP code remotely.
+
+        This module was tested against Navigate CMS 2.8.
+      ),
+      'Author' =>
+        [
+          'Pyriphlegethon' # Discovery / msf module
+        ],
+      'License' => MSF_LICENSE,
+      'References' =>
+        [
+          ['CVE', '2018-17552'], # Authentication bypass
+          ['CVE', '2018-17553']  # File upload
```

Comment:
Are these the correct CVEs? I could not find information about these.

Reply:
They're public now.